client/my-sites/site-settings/main.jsx

```diff
@@ -28,15 +27,13 @@ export class SiteSettingsComponent extends Component {
 	};
 
 	static defaultProps = {
-		section: 'general'
+		section: 'guidedTransfer'
```

Comment:
Yeah, this was on my immediate list after I land this PR. I didn't want to bloat it too much, though. If you feel it's right, I can address it in this PR.